### PR TITLE
Create portfolio-mobile header

### PR DESCRIPTION
**Pull request by lukman155**
# **Create a Portfolio: setup and mobile first**
## Learning Objectives

- Understand how to parse a Figma design to create a UI.
- Flexbox to place elements in the page.
- Build a personal portfolio site.
- Use images and backgrounds to enhance the look of the website.
## Description
In this project, I set up a new repository and prepared it for development using best practices (e.g. linters).
my goal here is to build the first 2 sections of the mobile website using the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) given .
##General requirements
- I made sure that there are no linter errors.
- I made sure that I used the correct GitHub flow.
- I made sure that I documented my work.
## Requirements
- I documented my work [in a professional way.
- I designed the page using responsible design principles.
- I opened a pull request.
# 🤗

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <div class="header"> 
+            <div class="headertext">Welcome</div>
+            <div>
+            <label for="trigger">
+            <input type="checkbox" class="trigger" id="trigger"/>
+            <div class="hamburger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
```
Comment (levy002) [inline index.html:25]: - [ ] Kindly make sure to wrap your hamburger icon into the `nav` tag in order to correctly format your HTML semantically. :wink:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Luk, A man</title>
+</head>
+<body>
+    <header>
+        <div class="header"> 
+            <div class="headertext">Welcome</div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap your logo into an `a` tag in order to increase web accessibility. :wink: You can read this [documentation](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                      have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    <div class="social">
+                        <a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a>
+                        <a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a>
+                        <a href="#"><img src="images\medium.svg" alt="medium"></a>
+                        <a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a>
+                        <a href="#"><img src="images\angellist.svg"></a>
+        
+                    </div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap your social media icons into `ul` tag and wrap each one into `li` as well in order to correctly format your HTML semantically. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @lukman155   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good UI :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- LInters are green :heavy_check_mark: 
- Good Readme description :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @lukman155 , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**